### PR TITLE
docs: update tutorial wrapping up

### LINKED ISSUE #269: Tutorial wrapping up page was reverted with website update
The wrapping up step for the React series was updated with the launch of the Vue series to remove the t-shirt section and point to the new badge application form. 

It looks as though the update to the new website missed this change.

### PR DESCRIPTION
**Pull request by mattrosno**
Closes #269

- Removes references to t-shirts as we're past those deadlines
- Cleans up some AI so it's easier to find the badge application forms

### CODE REVIEW

**Comment by mattrosno** [comment]
@connor-leech can you review, esp the subheadings?

File: src/pages/tutorial/react/wrapping-up.mdx [hunk 1]
```diff
@@ -4,19 +4,18 @@ tabs:
   ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
 ---
 
-### Thanks for completing our tutorial and helping us improve it along the way. As a way to say thanks, you can apply for an IBM Digital Badge and a free shirt!
+### Thanks for completing our tutorial and helping us improve it along the way. Want to show the world your new skills? Please apply for the IBM Digital Badge.
```
Comment (connor-leech) [inline src/pages/tutorial/react/wrapping-up.mdx]:
```suggestion
<PageDescription>

Thanks for completing the IBM Carbon tutorial and helping us improve it along the way. If you want to show the world your new skills, you can apply for an IBM Digital Badge.

<PageDescription>
```

File: src/pages/tutorial/react/wrapping-up.mdx [hunk 2]
```diff
@@ -4,19 +4,18 @@ tabs:
...
+<AnchorLink>Application form</AnchorLink>
+<AnchorLink>Badge information</AnchorLink>
 
 </AnchorLinks>
 
-## Badging
+## Application form
 
-Once you complete all five steps of the tutorial, you can apply for the IBM Digital Badge. These badges are available to anybody—not just IBM employees.
+Once you complete all five steps of the tutorial, you can [apply for the IBM Digital Badge](https://www.surveygizmo.com/s3/5163103/IBM-Carbon-Design-System-Developer-Essentials-React-v2). These badges are available to anybody — not just IBM employees.
```
Comment (connor-leech) [inline src/pages/tutorial/react/wrapping-up.mdx]:
```suggestion
Once you complete all five steps of the tutorial, you can apply for an [IBM Digital Badge](https://www.surveygizmo.com/s3/5163103/IBM-Carbon-Design-System-Developer-Essentials-React-v2). These badges are available to anybody — not just IBM employees.
```

File: src/pages/tutorial/react/wrapping-up.mdx [hunk 3]
```diff
@@ -28,18 +27,24 @@ Once you complete all five steps of the tutorial, you can apply for the IBM Digi
 </Column>
 </Row>
 
+Once you submit the application, please allow 2-4 days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (connor-leech) [inline src/pages/tutorial/react/wrapping-up.mdx]:
```suggestion
Once you submit the application, please allow two to four days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

File: src/pages/tutorial/vue/wrapping-up.mdx [hunk 4]
```diff
@@ -4,19 +4,18 @@ tabs:
   ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
 ---
 
-### Thanks for completing our tutorial and helping us improve it along the way. As a way to say thanks, you can apply for an IBM Digital Badge and a free shirt!
+### Thanks for completing our tutorial and helping us improve it along the way. Want to show the world your new skills? Please apply for the IBM Digital Badge.
```
Comment (connor-leech) [inline src/pages/tutorial/vue/wrapping-up.mdx]:
```suggestion
### Thanks for completing our tutorial and helping us improve it along the way. If you want to show the world your new skills, you can apply for an IBM Digital Badge.```

File: src/pages/tutorial/vue/wrapping-up.mdx [hunk 5]
```diff
@@ -4,19 +4,18 @@ tabs:
...
+<AnchorLink>Application form</AnchorLink>
+<AnchorLink>Badge information</AnchorLink>
 
 </AnchorLinks>
 
-## Badging
+## Application form
 
-Once you complete all five steps of the tutorial, you can apply for the IBM Digital Badge. These badges are available to anybody—not just IBM employees.
+Once you complete all five steps of the tutorial, you can [apply for the IBM Digital Badge](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2). These badges are available to anybody — not just IBM employees.
```
Comment (connor-leech) [inline src/pages/tutorial/vue/wrapping-up.mdx]:
```suggestion
Once you complete all five steps of the tutorial, you can apply for an [IBM Digital Badge](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2). These badges are available to anybody — not just IBM employees.
```

File: src/pages/tutorial/vue/wrapping-up.mdx [hunk 6]
```diff
@@ -28,19 +27,24 @@ Once you complete all five steps of the tutorial, you can apply for the IBM Digi
 </Column>
 </Row>
 
+Once you submit the application, please allow 2-4 days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```
Comment (connor-leech) [inline src/pages/tutorial/vue/wrapping-up.mdx]:
```suggestion
Once you submit the application, please allow two to four days for issuing the badge. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
```

File: src/pages/tutorial/vue/wrapping-up.mdx [hunk 7]
```diff
@@ -52,22 +56,6 @@ Acclaim Support: For questions related to your Acclaim badge earner account and
...
-</Column>
-</Row>
-
-## Application
-
-Complete the [application form](https://www.surveygizmo.com/s3/5135615/IBM-Carbon-Design-System-Developer-Essentials-Vue) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+### Application form
 
-_\* Shirts are available for a limited time while supplies last to U.S. shipping addresses and outside the U.S. to participating FED@IBM branches._
+To apply for the IBM Digital Badge, please use [this link](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2).
```
Comment (connor-leech) [inline src/pages/tutorial/vue/wrapping-up.mdx]:
```suggestion
To apply for the IBM Digital Badge, please complete the [form and survey](https://www.surveygizmo.com/s3/5215482/IBM-Carbon-Design-System-Developer-Essentials-Vue-v2).
```

File: src/pages/tutorial/react/wrapping-up.mdx [hunk 8]
```diff
@@ -51,22 +56,6 @@ Acclaim Support: For questions related to your Acclaim badge earner account and
...
-</Column>
-</Row>
-
-## Application
-
-Complete the [application form](https://www.surveygizmo.com/s3/5075530/Dev-Essentials-Badge-Application) to request the digital badge and t-shirt. Once you submit the application, please allow 2-4 days for issuing the badge and 4-6 weeks to receive the t-shirt. Once the badge is issued, you will be notified and must accept the badge via Acclaim.
+### Application form
 
-_\* Shirts are available for a limited time while supplies last to U.S. shipping addresses and outside the U.S. to participating FED@IBM branches._
+To apply for the IBM Digital Badge, please use [this link](https://www.surveygizmo.com/s3/5163103/IBM-Carbon-Design-System-Developer-Essentials-React-v2).
```
Comment (connor-leech) [inline src/pages/tutorial/react/wrapping-up.mdx]:
```suggestion
To apply for the IBM Digital Badge, please complete the [form and survey](https://www.surveygizmo.com/s3/5163103/IBM-Carbon-Design-System-Developer-Essentials-React-v2).
```

**Pull request merged**